src/test/ui/placement-syntax.rs

```diff
@@ -1,7 +1,8 @@
+// run-pass
+
 fn main() {
     let x = -5;
-    if x<-1 {
-    //~^ ERROR emplacement syntax is obsolete
+    if x<-1 { // ok: parses as a comparison
```

Comment:
So I think it's good that we remove it from the AST and such, but keeping options open re. `<-` and not introducing new stable behavior would be good so let's reject this in the parser?